data/avro-schema/src/main/resources/avro/net/corda/data/flow/output/FlowStatus.avsc

```diff
@@ -49,6 +50,17 @@
       "type": ["null", "net.corda.data.ExceptionEnvelope"],
       "doc": "Optional error message, this will be set if the flow status is 'FAILED'"
     },
+    {
+      "name": "processingTerminationDetails",
+      "type": [
+        "null",
+        {
+            "type": "map",
+            "values": "string"
+        }
+      ],
+      "doc": "Optional details about processing termination, for example, reasons and additional info about why a flow was killed."
+    },
```

Comment:
I think it is useful to see on the flow status why a flow was killed / why processing was terminated.

I have not exposed this on the REST endpoint, it's just on the avro message.